website/src/data/users.tsx

```diff
@@ -1538,12 +1538,21 @@ const Users: User[] = [
   },
   {
     title: 'Matej Jellus',
-    description: 'IT enthusiast that loves to write code, try new things and share knowledge. If not sitting in front of computer, then I am playing badminton, riding bike or hiking.',
+    description:
+      'IT enthusiast that loves to write code, try new things and share knowledge. If not sitting in front of computer, then I am playing badminton, riding bike or hiking.',
     preview: require('./showcase/juffalow.png'),
     website: 'https://juffalow.com/',
     source: 'https://github.com/juffalow/juffalow-com',
     tags: ['personal', 'opensource'],
   },
+  {
+    title: 'Signoz',
+    description: 'Open source Application Performance Monitoring (APM) & Observability tool. SigNoz helps developers monitor their applications & troubleshoot problems, an open-source alternative to DataDog, NewRelic, etc. 🔥 💻',
+    preview: require('./showcase/signoz.png'),
+    website: 'https://signoz.io/',
+    source: 'https://github.com/SigNoz/signoz',
```

Comment:
this does not seem to be the source of your Docusaurus site.
Can you update the source to your site source code, or remove the open-source tag?

Reply:
Hi @slorber 
Sorry i didn't into that. Right now site is not yet open source. probably i missed that part
 
Would it possible to follow this tag => 'Large Site', 'design' , 'personal' , 'product'

Let me know if this work for you
Thanks

Reply:
Yes, please do a followup to remove the `opensource` tag in that case. Sorry for missing that part...

Don't know if your site is `personal`, since `personal` and `product` are usually mutually exclusive. Considering `large site`, have you compared the scale of your site to others with this tag? I don't think your site is currently eligible although it's pretty close (200 pages)

Reply:
No currently we don't have 200 page. yeah i think personal would not work in this case thanks for the feedback

so raising the pr for 'product','design'

Reply:
Hi @Josh-Cena raised the [PR](https://github.com/facebook/docusaurus/pull/5949)